app/src/main/java/com/example/hm3_retrofit/adapter/ItemAdapter.kt

```diff
@@ -0,0 +1,75 @@
+package com.example.hm3_retrofit.adapter
+
+import android.content.ClipData
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hm3_retrofit.databinding.ItemLoadingBinding
+import com.example.hm3_retrofit.databinding.ItemPersonBinding
+import com.example.hm3_retrofit.model.ItemType
+
+
+class ItemAdapter(
+    private val onUserClicked: (ItemType) -> Unit,
```

Comment:
тогда уж так, ты же на элемент Loading нажать не можешь
```suggestion
    private val onUserClicked: (ItemType.CartoonPerson) -> Unit,
```